app/views/vacancies/_sorting_options.html.haml

```diff
@@ -1,8 +1,11 @@
 .govuk-grid-row.sortable-links
   .govuk-grid-column-one-half
     = form_tag('', method: :get) do
-      - search_criteria.each do |filter, value|
-        = hidden_field_tag filter, value
+      - search_criteria.each do |filter, value|  
+        - if filter == :phases
+          = hidden_field_tag "phases[]", value
```

Comment:
Does this work if phases has more than one value checked?

Reply:
Thanks for highlighting this. PR #1162 addresses this.